src/tests/operate.test.js

```diff
@@ -0,0 +1,38 @@
+import Big from 'big.js';
+import operate from '../components/logic/operate';
```

Comment:
[REQUIRED] The project requirements require that there's the usage of `Jest snapshots` for testing all the components. That means that there should be Jest snapshots for the `Button`, `Calculator`, `Home`, and `Quote` components.

Snapshot tests are a useful tool to ensure that your UI does not change unexpectedly. A typical snapshot test case renders a UI component, takes a snapshot, then compares it to a reference snapshot file stored alongside the test. The test will fail if the two snapshots do not match: either the change is unexpected, or the reference snapshot needs to be updated to the new version of the UI component. You read more about Jest snapshots [here](https://jestjs.io/docs/snapshot-testing).

Instead of rendering the graphical UI, which would require building the entire app, you can use a test renderer to quickly generate a serializable value for your React tree. 

Here's an example test for a Link component:

```
import renderer from 'react-test-renderer';
import Link from '../Link';

it('renders correctly', () => {
  const tree = renderer
    .create(<Link page="http://www.facebook.com">Facebook</Link>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
```

The first time this test is run, Jest creates a snapshot file that looks like this:

```
exports[`renders correctly 1`] = `
  <a
    className="normal"
    href="http://www.facebook.com"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    Facebook
  </a>
`;
```

The snapshot artifact should be committed alongside code changes and reviewed as part of your code review process. On subsequent test runs, Jest will compare the rendered output with the previous snapshot. If they match, the test will pass. If they don't match, either the test runner found a bug in your code (in this case, in the `<Link>` component) that should be fixed, or the implementation has changed and the snapshot needs to be updated.